### PR TITLE
{bio}[foss-2022b] SVclone 1.1.2 w/Python/3.10.8

### PR DESCRIPTION
**Pull request by fizwit**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by verdurin** [comment]
Test report by @verdurin
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
easybuild-c7.novalocal - Linux CentOS Linux 7.9.2009, x86_64, Intel Xeon Processor (Skylake, IBRS), Python 3.6.8
See https://gist.github.com/verdurin/3d27cb5f00f5eb2705bc1810d43670b9 for a full test report.

File: easybuild/easyconfigs/s/SVclone/SVclone-1.1.2-foss-2022b.eb [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+exts_list = [
+    ('pysam', '0.19.1', {
+        'checksums': ['dee403cbdf232170c1e11cc24c76e7dd748fc672ad38eb0414f3b9d569b1448f'],
+    }),
+    ('PyVCF3', '1.0.3', {
+        'modulename': 'vcf',
+        'checksums': ['4b16d71c8b97010487e2c939fb4d5707b7bbfa4e2b313df9dba3e372c5ba031d'],
+    }),
+    (name, version, {
+        'modulename': False,
```
Comment (verdurin) [inline easybuild/easyconfigs/s/SVclone/SVclone-1.1.2-foss-2022b.eb:33]: Haven't seen this usage before - could you explain @fizwit (as much for my benefit as anything else).

Comment (fizwit) [reply easybuild/easyconfigs/s/SVclone/SVclone-1.1.2-foss-2022b.eb:33]: Are you referring to the 'modulename': False?  At first it seemed that SVclone only installed to /bin, But know I see a python library and `import SVclone` works fine.   I will update the PR.

Comment (fizwit) [reply easybuild/easyconfigs/s/SVclone/SVclone-1.1.2-foss-2022b.eb]: FYI, Easybuild sanity check imports module names as %(namelower)s.  Need to implicitly use `'modulename': '%(name)s',` to pass sanity check.

Comment (smoors) [reply easybuild/easyconfigs/s/SVclone/SVclone-1.1.2-foss-2022b.eb]: no need to use the `'%(name)s'` template, can use `name` directly, done in https://github.com/easybuilders/easybuild-easyconfigs/pull/18084/commits/e1aa6c6e7102578e8ff4456d2a089b9cc122c3f3

**Comment by smoors** [comment]
@boegelbot please test @ jsc-zen2

**Review by smoors — APPROVED** [review]
lgtm

**Comment by smoors** [comment]
Going in, thanks @fizwit!

**Pull request merged**